automation/configs/process.yml

```diff
@@ -0,0 +1,9 @@
+instances: 1
```

Comment:
Can you explain a little what all these yml files are for and how they are being utilized.

I'm assuming they spin up to do some processing then get taken down. Is there any worry of shared resources being streched too thin?

Reply:
Each is a different configuration for the automation app. It can only use one at a time. and each is only on for just long enough to do its assigned task.

no, that is why they are only on for just long enough to do their task then get shut down.